### ISSUE TITLE
Autofill for openstack horizon does not work

### ISSUE THREAD

**Issue by jsm222** [passbolt/passbolt_browser_extension#71]
Auto-fill for openstack horizon does not work
* Passbolt Version: 2.9.2
* Platform and Target:
-- Browser: Firefox 66.0.3+build1-0ubuntu0.18.04.
-- Web server: nginx 
The reason is that openstack dashboard contains the following before the actual login form
(edting it out with developer tools and auto-fill works.  Setting the horizon setting disable_password_reveal to True does not remove the fake_credentials div, and thus is not a fix. What could a solution be... Perhaps fil out all occurences of username and password fields??

```
<div class="fake_credentials has-feedback" style="display: none">
          <input type="text" name="fake_email" value="">
          <input type="password" name="fake_password" value=""><span class="form-control-feedback fa fa-eye password-icon"></span>
        </div>
```

**Comment by cedricalfonsi** [comment]
Thank you for your report @jsm222 
This info will help us to improve the aufill

**Comment by stripthis** [comment]
Fixed with v2.10.2
